analyzer/src/test/kotlin/PackageManagerTest.kt

```diff
@@ -19,9 +19,11 @@
 
 package org.ossreviewtoolkit.analyzer
 
-import io.kotest.matchers.shouldBe
```

Comment:
Should this commit also turn the uses of `shouldContainExactly` into `should contrainExcatly`? See e.g. `ScopeTest.kt`.

Reply:
I was only searching for `shouldBe listOf`, maybe this could be done in a separate commit later where also `shouldContainExactlyInAnyOrder` with `should containExactlyInAnyOrder` and similar. My focus was more on using matchers instead of the generic `shouldBe` where possible.